README.adoc

```diff
@@ -1,3 +1,23 @@
+=== ⚠️ CAUTION
+
+The developer team is currently preparing the release of OCaml 5.0. This release
+sports a full rewrite of its runtime system for shared-memory parallel
+programming using domains and native support for concurrent programming using
+effect handlers.
+
+The initial release of OCaml 5.0 will only support the native compiler under
+ARM64 and x86-64 architectures under Linux and macOS. On Windows, only the
+Mingw-w64 port is supported.  Support for other 64-bit architectures and
+systems, including Windows, will be added back in later releases. On 32-bit
```

Comment:
I don't think "including Windows" is needed here?

Reply:
Indeed, will remove. Do you think the "On Windows, only ... is supported" should be left here if we are doing a specific paragraph explaining the situation on Windows in the other README?

Reply:
I think it's good to have that sentence here, yes, as it's displayed on the GitHub UI when you visit the repo, etc. (i.e. it's displayed _before_ you go to visit README.win32.adoc)

Reply:
Thanks, removed "including Windows" as suggested.